docker/compose/validator-testnet/docker-compose.yaml

```diff
@@ -24,42 +24,45 @@ services:
             - type: volume
               source: diem-shared
               target: /opt/diem/var
+            - type: bind
+              source: ./validator_node_template.yaml
+              target: /opt/diem/var/validator_node_template.yaml
         command: ["/opt/diem/bin/diem-node", "--test", "--config", "/opt/diem/var"]
         ports:
             - "8080:8080"
 
     faucet:
-      image: "libra/faucet:${IMAGE_TAG:-devnet}"
-      depends_on:
-        - validator
-      networks:
-        shared:
-      volumes:
-        - type: volume
-          source: diem-shared
-          target: /opt/diem/var
-      command: >
-        /bin/bash -c "
-            for i in {1..10}
-            do
-                if [[ -s /opt/diem/var/mint.key ]]
-                then
-                    /opt/diem/bin/diem-faucet \
-                        --address 0.0.0.0 \
-                        --port 8000 \
-                        --chain-id TESTING \
-                        --mint-key-file-path /opt/diem/var/mint.key \
-                        --server-url http://validator:8080/v1
-                    exit $$?
-                else
-                    echo 'Validator has not populated mint.key yet. Is it running?'
-                    sleep 1
-                fi
-            done
-            exit 1
-          "
-      ports:
-        - "8000:8000"
+        image: "libra/faucet:${IMAGE_TAG:-devnet}"
+        depends_on:
+            - validator
+        networks:
+            shared:
+        volumes:
+            - type: volume
+              source: diem-shared
+              target: /opt/diem/var
+        command: >
+            /bin/bash -c "
+                for i in {1..10}
+                do
+                    if [[ -s /opt/diem/var/mint.key ]]
+                    then
+                        /opt/diem/bin/diem-faucet \
+                            --address 0.0.0.0 \
+                            --port 8000 \
+                            --chain-id TESTING \
+                            --mint-key-file-path /opt/diem/var/mint.key \
+                            --server-url http://validator:8080/v1
+                        exit $$?
+                    else
+                        echo 'Validator has not populated mint.key yet. Is it running?'
+                        sleep 1
+                    fi
+                done
+                exit 1
+            "
+        ports:
+            - "8000:8000"
```

Comment:
I don't think you intended for all of these whitespace changes?

Reply:
There was a white space inconsistency in the file. I hadn't realized I added to it in the first commit. I think the original was 4 spaces and a recent tweak made it both 2 and 4. Let me look up the recommendation and swap the ordering of my commits.

Reply:
I think we actually used 4 everywhere, but 2 is the recommended and looks a little cleaner considering the lists, so I at least adjusted for the yamls in this commit.